iotdb-1.3/src/main/java/cn/edu/tsinghua/iot/benchmark/iotdb130/IoTDB.java

```diff
@@ -83,33 +83,20 @@
 import java.util.stream.Collectors;
 
 /** this class will create more than one connection. */
-public class IoTDB implements IDatabase {
+public class IoTDB extends RegisterSchema {
```

Comment:
I think we need to consider the name of class under iotdb-1.3, for example PayLoad -> IoTDBRestPayLoad maybe better? At the same time, consider QueryResult、RegisterSchema and RestAPI